nevergrad/benchmark/experiments.py

```diff
@@ -23,6 +23,7 @@
 from nevergrad.functions.powersystems import PowerSystem
 from nevergrad.functions.stsp import STSP
 from nevergrad.functions.rocket import Rocket
+from nevergrad.functions.gym import GymMulti  # mypy: ignore
```

Comment:
should/must not be necessary
```suggestion
from nevergrad.functions.gym import GymMulti
```